### DISCUSSION TITLE
Why is large-v2 Slow? is it the Better model after all?

### DISCUSSION THREAD

**Discussion by NormalFall0** [openai/whisper#2174] [Q&A]
Hi
I noticed large v3 (large) to have lot of hallucinations.
v2 does not have them. But... IT IS SLOW.

Is that on purpose? As long as it transcribes correclty I don't complain! I was just wondering if we can keep the EXACT quality as large v2 and make it faster perhaps? Was it not optimized with cuda or other accelarations for some reason? Mayeb that has no link

**Comment by phineas-pta** [comment]
either get better hardware (gpu)

or use optimized forks like `faster-whisper` or `whisper.cpp`
